scripts/ipfs/install

```diff
@@ -23,8 +23,8 @@ rm -rf "$BASE_DIR/tmp"
 ipfs init || true
 
 # Configure HTTP to IPFS gateway
-# shellcheck source=../shared/nginx/install
-source "$BASE_DIR/../shared/nginx/install"
+# shellcheck source=../nginx/install
+source "$BASE_DIR/../nginx/install"
```

Comment:
Should no longer be installed as part of the module remove line 26, 27

already dont in main script
https://github.com/tomeshnet/prototype-cjdns-pi/pull/252/files#diff-70643e83d82dc35762cb692e2eb6e37cR335

Reply:
Of course, all these installs. I'll fix that.